src/libertem/web/jobs.py

```diff
@@ -95,6 +98,14 @@ def get_analysis_by_type(self, type_):
         self.finish()
         self.event_registry.broadcast_event(msg)
 
+        if hasattr(analysis, 'controller'):
+            return await analysis.controller(
+                cancel_id=uuid, executor=executor,
+                job_is_cancelled=lambda: self.data.job_is_cancelled(uuid),
+                send_results=lambda results, finished: self.send_results(results, uuid,
+                 finished=finished)
```

Comment:
Nitpick: `finished` should be aligned with the other params (yeah, sorry, my OCD is kicking in :grin:)